report/report-generator.js

```diff
@@ -30,17 +30,21 @@ class ReportGenerator {
   /**
    * Returns the report HTML as a string with the report JSON and renderer JS inlined.
    * @param {LH.Result} lhr
+   * @param {string=} reportTemplate
+   * @param {string=} reportJs
    * @return {string}
    */
-  static generateReportHtml(lhr) {
+  static generateReportHtml(lhr,
```

Comment:
little uncomfortable with making this part of the report generator api for just a test file. wb just duplicating this function (it's pretty simple, although the sanitization could be pulled out) and calling `ReportGenerator.replaceStrings` directly from `lighthouse-core/scripts/build-report-for-autodeployment.js` ?